invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""MARCXML based Schema for Invenio RDM Records."""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import bleach
+from invenio_access.permissions import system_identity
+from invenio_vocabularies.proxies import current_service as vocabulary_service
+from marshmallow import Schema, fields, missing
+
+from ..ui.schema import current_default_locale
+from ..utils import get_vocabulary_props
+
+
+class MARCXMLSchema(Schema):
+    """Schema for records in MARC."""
+
+    contributors = fields.Method("get_contributors")
```

Comment:
How was this list gathered? From zenodo? Is there something missing still from InvenioRDM model?

Reply:
Ok so most of the schema is taken from
https://github.com/TLGINO/invenio-rdm-records/blob/d1052925f1106be65c80d4c94d91d3b31572837d/invenio_rdm_records/resources/serializers/dublincore/schema.py#L19
I have adapted/omited some to better correspond to the marcxml standard